File: AppBackend/AvioCarBackend/AvioCarBackend/Model/BranchOffice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace AvioCarBackend.Model
{
    /// <summary>
    /// BranchOfficeID -------------- indetifikator filijale
    /// BranchOfficeAddress --------- adresa filijale
    /// City ------------------------ grad
    /// Country --------------------- drzava
    /// RentACarService ------------- rent-a-car servis kome filijala pripada
    /// </summary>
    [Table("BranchOffices")]
    public class BranchOffice
    {
        public int BranchOfficeID { get; set; }
        public string BranchOfficeAddress { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public RentACarService RentACarService { get; set; }
    }
}

File: AppFrontend/AvioCarFrontend/src/app/entities/car-entities/branch-office/branch-office.ts
export class BranchOffice {
    branchOfficeID: string;
    branchOfficeAddress: string;
    city: string;
    country: string;

    constructor(branchOfficeID: string, branchOfficeAddress: string, city: string, country: string){
        this.branchOfficeID = branchOfficeID;
        this.branchOfficeAddress = branchOfficeAddress;
        this.city = city;
        this.country = country;
    }
}
